jetty/src/main/scala/org/http4s/server/jetty/JettyBuilder.scala

```diff
@@ -3,7 +3,7 @@ package server
 package jetty
 
 import com.codahale.metrics.{InstrumentedExecutorService, MetricRegistry}
-import com.codahale.metrics.jetty9.{InstrumentedQueuedThreadPool, InstrumentedConnectionFactory}
+import com.codahale.metrics.jetty9.{InstrumentedQueuedThreadPool, InstrumentedHandler, InstrumentedConnectionFactory}
```

Comment:
Hmmm, I'm still getting the error on a fresh checkout.

    $git log --pretty=oneline --abbrev-commit
    39d674a Nope. It's still a snapshot.

    $ sbt clean test
    [info] Passed: Total 27, Failed 0, Errors 0, Passed 27
    [error] (jetty/compile:compileIncremental) Compilation failed
    [error] Total time: 149 s, completed 24/06/2015 11:46:22 AM